tethysext/atcore/public/resource_workflows/report_workflow_results_view.js

```diff
@@ -19,32 +81,16 @@ $(function() {
                     tipLabel: 'North Up',
                 }),
             ]),
-            layers: [
-                new ol.layer.Tile({
-                    source: new ol.source.XYZ({
-                        attributions: 'Tiles © <a href="https://services.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer">ArcGIS</a>',
-                        url: 'https://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
-                    }),
-                }),
-                new ol.layer.Tile({
-                    source: new ol.source.XYZ({
-                        attributions: 'Tiles © <a href="https://services.arcgisonline.com/ArcGIS/rest/services/Reference/World_Boundaries_and_Places/MapServer">ArcGIS</a>',
-                        url: 'https://server.arcgisonline.com/ArcGIS/rest/services/Reference/World_Boundaries_and_Places/MapServer/tile/{z}/{y}/{x}',
-                    }),
-                }),
-                new ol.layer.Image({
-                    source: wmsSources[i],
-                }),
-            ],
+            layers: layers,
             target: `openLayerMap${i+1}`,
             view: new ol.View({
-                center: ol.proj.fromLonLat([($(obj).data("layer-extent")[0] + $(obj).data("layer-extent")[2])/2, ($(obj).data("layer-extent")[1] + $(obj).data("layer-extent")[3])/2]),
+                center: ol.proj.fromLonLat([(layer_extent[0] + layer_extent[2])/2, (layer_extent[1] + layer_extent[3])/2]),
                 zoom: 12,
             }),
         });
 
         // Find wms layer extent and fit the map to it.
-        extents[i] = ol.proj.transformExtent($(obj).data("layer-extent"), 'EPSG:4326', 'EPSG:3857');
+        extents[i] = ol.proj.transformExtent(layer_extent, 'EPSG:4326', 'EPSG:3857');
```

Comment:
I think this can be simplified to `extent = ...;` Is there a need to store all the extents in the `extents` array?